try {
    some();
} catch (e) {
    console.log("d", e);
}